small typo fixed

castaned opened the pull request.

dildick commented:
@castaned I see a lot more changes than just a typo getting fixed. What are the other changes?

castaned commented:
@dildick   It looks like a lot but actually I just removed part of the code that was duplicated, I introduced this when copying from the HitRecovery analyzer to the cutFlowanalyzer. this part was not interfering with the compilation. in any case it is fixed.  The rest is just ordering some initializations in a different way.
